src/libs/actions/Policy.ts

```diff
@@ -129,6 +129,14 @@ type WorkspaceMembersRoleData = {
     role: typeof CONST.POLICY.ROLE.ADMIN | typeof CONST.POLICY.ROLE.USER;
 };
 
+const CURRENCY_AU = 'AUD';
+const CURRENCY_CA = 'CAD';
+const CURRENCY_GB = 'GBP';
+const CURRENCY_US = 'USD';
+const CURRENCY_EUR = 'EUR';
+
+const DIRECT_REIMBURSEMENT_CURRENCIES: string[] = [CURRENCY_AU, CURRENCY_CA, CURRENCY_EUR, CURRENCY_GB, CURRENCY_US];
+
```

Comment:
We already have these defined [here](https://github.com/Expensify/App/blob/8af0109d7e9ce0ee583dbb4a8cea13c24fb5fcf5/src/CONST.ts#L496-L506), it also seems like we support NZD, which is missing in your list above

Reply:
+1 definitely lets reuse this from Const or add there what is missing there

Reply:
> We already have these defined [here](https://github.com/Expensify/App/blob/8af0109d7e9ce0ee583dbb4a8cea13c24fb5fcf5/src/CONST.ts#L496-L506), it also seems like we support NZD, which is missing in your list above

NZD is not supported for direct reimbursement.

See [here](https://github.com/Expensify/PHP-Libs/blob/c3bd9f22ba3de58d510fc1d11cb31dea267d93ad/src/Constants.php#L46-L52) and [here](https://github.com/Expensify/Web-Expensify/blob/ae3c08686fb72664fe69356ba9dacc6313e4e756/lib/PolicyAPI.php#L3726-L3738)